style.css

```diff
@@ -0,0 +1,189 @@
+  padding: 2.5rem;
+  align-items: center;
+}
+
+.navbar__tittle {
+  font-family: "Fraunces", serif;
+  font-size: 1.5rem;
+  font-weight: 600;
+  letter-spacing: -1px;
+  color: var(--dark-grey-blue);
+}
+
+.navbar__menu__list--off {
+  display: none;
+  list-style: none;
+}
+
+.navbar__menu__icon {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  width: 1.5rem;
+  height: 2rem;
+}
+
+.navbar__menu__icon--content {
+  background-color: var(--dark-grey-blue);
+  width: 100%;
+  height: 4px;
+  border-radius: 5px;
+}
+
+.navbar__menu__icon--content::after,
+.navbar__menu__icon--content::before {
+  content: "";
+  width: 100%;
+  height: 4px;
+  background-color: var(--dark-grey-blue);
+  position: absolute;
+  border-radius: 5px;
+}
+
+.navbar__menu__icon--content::after {
+  top: 20%;
+}
+
+.navbar__menu__icon--content::before {
+  bottom: 20%;
+}
+
+.navbar__menu__icon--content--active {
+  background: none;
+}
+
+.navbar__menu__icon--content--active::after {
+  transform: translateY(-50%) rotate(45deg);
+  top: 50%;
+}
+
+.navbar__menu__icon--content--active::before {
+  transform: translateY(50%) rotate(-45deg);
+  bottom: 50%;
+}
+
+.navbar__menu__list--on {
+  display: flex;
+  position: absolute;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  padding-top: 10rem;
+  background: linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, 1) 60%,
+    rgba(255, 255, 255, 0.5)
+  );
+  flex-direction: column;
+  gap: 3rem;
+  font-family: "Fraunces", serif;
+  font-weight: 600;
+  font-size: 2rem;
+  color: var(--dark-grey-blue);
+}
+
+.navbar__menu__list--on .navbar__list--item::first-letter {
+  text-transform: uppercase;
+}
+
+.navbar__logo-container {
+  z-index: 1;
+}
+
+@media (min-width: 768px) {
+  .navbar__menu__icon {
+    display: none;
+  }
+  .navbar__menu__list--off {
+    display: flex;
+    gap: 1rem;
+    text-transform: uppercase;
+    font-size: 0.9rem;
+    color: var(--grey);
+    font-family: "Barlow", sans-serif;
+    font-weight: 600;
+  }
+  .navbar__tittle {
+    font-size: 2rem;
+  }
+  .navbar__iamge {
+    width: 1.5rem;
+  }
+}
+
+@media (min-width: 1440px) {
+  .navbar {
+    padding-left: 4rem;
+    padding-right: 4rem;
+  }
+}
+
+/*HERO*/
+
+.hero-container {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 2rem 1rem 2rem;
+  padding: 7rem 1.5rem 7rem 1.5rem;
+  padding-bottom: 7rem;
+  background-color: red;
+  gap: 2rem;
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: url(images/Hero-bg.png);
+  background-size: cover;
+}
+
+.hero-container__tittle {
+  font-size: 3rem;
+  font-family: "Fraunces", serif;
+  letter-spacing: -2px;
+  color: var(--white);
+}
+
+.hero-container__content {
+  font-family: "Barlow", serif;
+  color: var(--light-cream);
+  opacity: 80%;
+}
+
+.hero-container__button {
+  width: 80%;
```

Comment:
Set button width and height with padding. Here you set button width to 80% and this is what it looks like to me: 
![image](https://github.com/ksawerykasp/coffeeroasters/assets/76774237/07516e2c-dd41-46c5-a078-acc615e13ddb)
Button has 800px